README.md

```diff
@@ -1 +1,15 @@
-# phpstorm-gh-pr-discussion-bug
\ No newline at end of file
+# phpstorm-gh-pr-discussion-bug
+
+## What steps will reproduce the problem?
```

Comment:
This is:

regular text

*italic*

**bold**

> quote

* unordered
* list

1. ordered
2. list

```
code block
```

Inline `code` sample

| table |
| ----- |
| row |